coala_quickstart/generation/Bears.py

```diff
@@ -37,7 +36,7 @@ def filter_relevant_bears(used_languages,
         A dict with language name as key and bear classes as value.
     """
     args = arg_parser.parse_args() if arg_parser else None
-    log_printer = LogPrinter(NullPrinter())
+    log_printer = None
```

Comment:
@seeeturtle The CIs are failing due to the changes, do all the tests pass in your local computer?

Reply:
@satwikkansal , no they are failing because the repo has two build failures on master.  See two issues raised